### ISSUE TITLE
Templater button loses template in button block after first use

### ISSUE THREAD

**Issue by DanF-88** [shabegom/buttons#38]
If I have a button block like 
```button
name Create New Meeting File
type note(Team meeting <% tp.date.now("YYYY-MM-DD") %>) template
action Meeting template file
templater true
```
^button-myteammeeting

If I press the button, the file ('Team meeting 2021-05-14') is successfully created, but the button block now reads:

```button
name Create New Meeting File
type note(Team meeting 2021-05-14) template
action Meeting template file
templater true
```
^button-myteammeeting

**Comment by shabegom** [comment]
Oh no! That's not supposed to be happening. Kind of defeats the point eh? I'll take a look. 

Some troubleshooting questions:
- version of obsidian, templater & buttons
- what else is in the note with the button?
- anything logged in the dev console?
- is there any difference if you add the `split` arg to note?

**Comment by DanF-88** [comment]
I’m running Obsidian v0.12.3
Buttons 0.4.5
Templater 1.6.0
MacOS 11.3.1
(All latest for this date)

Same problem occurs if the split parameter is present

I have text before and after the button but no other blocks.

Here’s the log when I run it without a split argument:
main.ts:121 TRIGGERED TRIGGERED
buttonTypes.ts:226 Uncaught (in promise) TypeError: Cannot read property 
'split' of undefined
     at eval (eval at <anonymous> (app.js:1), <anonymous>:1291:69)
eval @ buttonTypes.ts:226
setTimeout (async)
templater @ buttonTypes.ts:221
async function (async)
templater @ buttonTypes.ts:212
clickHandler @ button.ts:49
async function (async)
clickHandler @ button.ts:43
eval @ button.ts:31
s @ app.js:1

**Comment by shabegom** [comment]
Excellent I know what the problem is. Now I just need to work out a solution. Will ship a fix as soon as I can.

**Comment by shabegom** [comment]
@DanF-88 I've just released 0.4.6 and this bug should now be fixed. Please give it a ty and let me know!
